demo/admin/src/pages/errorpage/ErrorPage.tsx

```diff
@@ -0,0 +1,80 @@
+import "@vivid/roboto-font";
```

Comment:
pls remove roboto-font and `kwfUp-roboto`, and use `@fontsource/roboto` instead like mui: https://mui.com/material-ui/react-typography/#general